bin/steps/python

```diff
@@ -41,7 +41,11 @@ esac
 
 # The Python runtime archive filename is of form: 'python-X.Y.Z-ubuntu-22.04-amd64.tar.zst'
 # The Ubuntu version is calculated from `STACK` since it's faster than calling `lsb_release`.
-UBUNTU_VERSION="${STACK/scalingo-}.04"
+# Our approach is a bit different than Heroku's, because we want to be more
+# generic and keep compatibility with STACK values such as `heroku-22` or
+# `scalingo-22-minimal`:
+UBUNTU_VERSION="${STACK/-minimal}"
+UBUNTU_VERSION="${UBUNTU_VERSION: -2}.04"
```

Comment:
question: are you sure about the white space after the `:`? If yes, then LGTM

Reply:
Yes, I'm sure :)
I even made a small script to test:

```bash
#!/usr/bin/env zsh

for stck in "scalingo-20" "scalingo-22" "scalingo-20-minimal" "scalingo-22-minimal"; do
    ub="${stck/'-minimal'}"
    ub="${ub: -2}"
    echo "${stck} --> ${ub}"
done
```